curp/tests/common/curp_group.rs

```diff
@@ -299,6 +299,40 @@ impl CurpGroup {
             .map(|(k, v)| (*k, vec![v.clone()]))
             .collect()
     }
+
+    pub async fn fetch_cluster_info(&self, addrs: &[String]) -> ClusterInfo {
```

Comment:
These two method seems not use in this pr. Shall we keep them?

Reply:
The ignored test use this two function, it will be enable after #448 merged